src/app/wishlists/wishlists.ts

```diff
@@ -38,14 +38,14 @@ let inventoryRolls: { [key: string]: InventoryWishListRoll } = {};
 export function getInventoryWishListRolls(
   allItems: DimItem[],
   rollsByHash: { [itemHash: number]: WishListRoll[] }
-): { [key: string]: InventoryWishListRoll } {
```

Comment:
Now the wishlist rolls can return undefined if there aren't any, instead of using the is enabled selector